environment/.deployment/dev/docker-compose.dev.yml

```
@@ -0,0 +1,76 @@
+networks:
+  traefik-proxy:
+    external: true
+  lmt-dev:
+    driver: bridge
+
+volumes:
+  lmt-dev-postgres-data:
+    name: lmt-dev-postgres-data
+
+services:
+  lmt-dev-app:
+    image: registry.blumilk.pl/internal-public/lmt:dev
+    container_name: lmt-dev-app-container
+    pull_policy: always
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "50m"
+        max-file: "5"
+    deploy:
+      mode: replicated
+      replicas: 1
+      resources:
+        limits:
+          memory: 1GB
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.lmt-dev-app.rule=Host(`${LMT_HOST_NAME}`)"
+      - "traefik.http.routers.lmt-dev-app.entrypoints=websecure"
+      - "traefik.http.routers.lmt-dev-app.tls=true"
+      - "traefik.http.routers.lmt-dev-app.tls.certresolver=lets-encrypt-resolver"
+      - "traefik.http.routers.lmt-dev-app.middlewares=response-gzip-compress@file"
```

```suggestion
      - "traefik.http.routers.lmt-dev-app.middlewares=no-index-robots-response-header@file,security-headers@file"
```

It is important to add these middlewares for security headers and no index by robots.

---

Updated middlewares.